.github/workflows/release_version.yaml

```diff
@@ -0,0 +1,128 @@
+name: Release tag version
+
+on:
+  workflow_dispatch:
+    branches:
+      - feature/*
+    inputs:
+      version:
+        required: true
+      description:
+        required: true
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+
+      - name: Setup project
+        run: make clean
+
+      - name: Tests - Unit
+        run: make test
+
+  release:
+    runs-on: ubuntu-latest
+    needs: build-test
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get credentials
+        uses: leanix/secrets-action@master
+        with:
+          secret-store-credentials: ${{ secrets.INJECTED_SECRET_STORE_CREDENTIALS }}
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+
+      - name: Login to Azure Container Registry
+        uses: azure/docker-login@v1
+        with:
+          login-server: 'leanixacr.azurecr.io'
+          username: ${{ env.ACR_USERNAME }}
+          password: ${{ env.ACR_PASSWORD }}
+
+      - name: Validate version input
+        run: echo ${{ github.event.inputs.version }}
+
+      - name: Extract base version
+        env:
+          SEM_VERSION: ${{ github.event.inputs.version }}
+        run: |
+          IFS="."
+          read -ra base_version <<< "$SEM_VERSION"
+          echo "BASE_VERSION=$base_version" >> $GITHUB_ENV
+
+      - name: Create github tag for semantic version
+        uses: rickstaa/action-create-tag@v1
+        with:
+          tag: ${{ github.event.inputs.version }}
+          message: ${{ github.event.inputs.description }}
+      - name: Create/Update github tag for latest version
+        uses: rickstaa/action-create-tag@v1
+        with:
+          tag: ${{ env.BASE_VERSION }}.0.0-latest
+          message: ${{ github.event.inputs.description }}
+          force_push_tag: true
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.8.1
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.4.0
+        with:
+          charts_dir: '/helm'
+          charts_repo_url: 'https://raw.githubusercontent.com/leanix/leanix-k8s-connector/main/helm'
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      - name: GoReleaser Release
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          version: v1.9.0
+          args: release --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GORELEASER_CURRENT_TAG: ${{ github.event.inputs.version }}
+
+      - name: Revert local changes and update helm repo for pull request
```

Comment:
Dint understand what this step does? What does it revert?

Reply:
When executing the go releaser it will add the binaries to the repository. We don't want those committed so that is why I revert the changes first.

Reply:
oh okay. Got it. but then cant we do a go-releaser step at the last?

Reply:
Will try

Reply:
Ok I can't move that but I also noticed, that I do not revert changes any more. That was an outdated message 👍